recipe/meta.yaml

```diff
@@ -22,27 +22,32 @@ requirements:
     run:
         - python
         - numpy x.x
-        - scipy
+        - scipy  # 0.16.0
```

Comment:
@dbuscombe-usgs `setup.py` limits to `scipy 0.16.0`, is that a hard limit or just something that is known to work? The problem is that there is no `scipy 0.16.0` compiled with the latest `numpy`, which would force a lot of packages to downgrade.